openapi/internal/jsoac/minimum.go

```diff
@@ -0,0 +1,12 @@
+package jsoac
+
+import (
+	schema "github.com/jsightapi/jsight-schema-core"
+)
+
+func newMinimum(astNode schema.ASTNode, t OADType) *Example {
```

Comment:
It is better to return a reference to the **Number** structure from this function